src/index.tsx

```diff
@@ -1,5 +1,6 @@
 import ReactDOM from 'react-dom';
 import { App } from './App';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
To fix the Github tests error you need to include `package.json` and `package-lock.json` files

![Screenshot 2023-04-06 at 11 30 48](https://user-images.githubusercontent.com/54940595/230321854-9aa041b9-45e6-48e0-8472-059ca77a4bb9.png)